tests/src/services/users.service.test.ts

```diff
@@ -123,7 +156,7 @@ describe('UsersService tests', () => {
   });
 
   describe('Cancelling user subscription', () => {
-    describe('Cancel the user individual subscription', () => {
+    describe('cancelUserIndividualSubscriptions()', () => {
```

Comment:
This goes directly against what we talked about in the last review. Talk about the use case, not about the function literally